android/envoy/src/main/kotlin/org/greatfire/envoy/NetworkIntentService.kt

```diff
@@ -964,11 +965,13 @@ class NetworkIntentService : IntentService("NetworkIntentService") {
                     Log.e(TAG, "config file error: " + e.localizedMessage)
                 } catch (e: Exception) {
                     Log.e(TAG, "unexpected error when reading file: " + e.localizedMessage)
+                    e.printStackTrace()
```

Comment:
why this function? i think it doesn't play well with logcat, also it probably doesn't filter out messages in prod builds, is that the intent?